harmony/config.py

```diff
@@ -15,9 +15,10 @@
 
 from dotenv import load_dotenv
 
-Environment = Enum('Environment', ['SBX', 'SIT', 'UAT', 'PROD'])
+Environment = Enum('Environment', ['LOCAL', 'SBX', 'SIT', 'UAT', 'PROD'])
 
 HOSTNAMES = {
+    Environment.LOCAL: 'localhost:3000',
```

Comment:
Is this something we can override? For example I tend to run my harmony on port 5000 locally.

Reply:
I was going to make it configurable and then thought YAGNI, but with the sneaking suspicion that we would need it. Yep, I'll add this.

Reply:
I run Harmony on a different port too.